src/services/events/leafletLabelEvents.js

```diff
@@ -0,0 +1,43 @@
+angular.module("leaflet-directive")
+.factory('leafletLabelEvents', function ($rootScope, $q, $log, leafletHelpers, leafletEventsHelpers) {
```

Comment:
labelEvents gets its own space to be shared among MarkersHelpers and PathHelpers